brightnessd.c

```diff
@@ -1017,6 +1030,80 @@ static uint8_t event_loop(struct Tglobalstate *pglobalstate, struct Txcb *pxcb,
     }
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// parse_uint8_t
+///////////////////////////////////////////////////////////////////////////////
+/** Converts a string to an uint8_t.
+
+    @param input            the string which should be converted
+    @param output           a pointer in which the conversion result will be written
+    @return                 a non-zero value means the conversion has failed
+*/
+static int parse_uint8_t(char* input, uint8_t* output) {
+    char *end = NULL;
+    errno = 0;
+
+    long temp = strtol(input, &end, 10);
+    if (end != input && errno != ERANGE && temp >= 0 && temp <= UINT8_MAX) {
+        *output = (uint8_t)temp;
+        return 0;
+    }
+    ERROR("[parse_uint8_t] Unable to convert %s to uint8_t\n", input);
+    return 1;
+}
+
+///////////////////////////////////////////////////////////////////////////////
+// print_usage
+///////////////////////////////////////////////////////////////////////////////
+void print_usage(void) {
+    printf("Usage: brightnessd [options...]\n"
+           "\n"
+           "Available options:\n"
+           "  --cycle-brightness   PERCENTAGE               Screen brightness percentage on cycle event (X11)\n"
+           "  --timeout-brightness PERCENTAGE               Screen brightness percentage on timeout event (X11)\n"
+           );
+}
+
+///////////////////////////////////////////////////////////////////////////////
+// parse_args
+///////////////////////////////////////////////////////////////////////////////
+/** Parse command-line argumens.
+
+    @param argc           the string which should be converted
```

Comment:
Hm, these comments don't really apply here, don't they?

Reply:
Oops. copy&paste mistake. Fixed (and renamed variables).